From 0f25027c68d890faf8ca44b6166ed49cef81e4d3 Mon Sep 17 00:00:00 2001
From: Kody A Kantor <kody@kkantor.com>
Date: Tue, 13 Nov 2018 20:37:32 +0000
Subject: [PATCH] MANTA-4018 manta-login tool should sort by shard number
 MANTA-4027 sdc-manta layout test needs an extra space in 'garbage-collector'
 row

---
 bin/manta-login        |  5 ++---
 test/tst.layout.js.out | 16 ++++++++--------
 2 files changed, 10 insertions(+), 11 deletions(-)

diff --git a/bin/manta-login b/bin/manta-login
index bfc2a1e..f83346c 100755
--- a/bin/manta-login
+++ b/bin/manta-login
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2016, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -53,8 +53,7 @@ elif [[ "$1" == "-h" || "$1" == "--help" ]]; then
 fi
 
 IFS=$'\n'
-results=($(manta-adm show -H -o service,shard,zonename,gz_admin_ip | \
-    grep $1 | sort))
+results=($(manta-adm show -H -o service,shard,zonename,gz_admin_ip | grep $1))
 if [[ ${#results[@]} -gt 1 ]]; then
     c=0
     if [[ -n $2 ]]; then
diff --git a/test/tst.layout.js.out b/test/tst.layout.js.out
index 47cf6f9..60c01e7 100644
--- a/test/tst.layout.js.out
+++ b/test/tst.layout.js.out
@@ -230,7 +230,7 @@ summary:
      marlin               -               32
      reshard              -                 
      pgstatsmon           -                 
-     garbage-collector    -                 
+     garbage-collector     -                 
      postgres             1                3
      postgres             2                3
      postgres             3                3
@@ -613,7 +613,7 @@ summary:
      marlin               -               16
      reshard              -                 
      pgstatsmon           -                 
-     garbage-collector    -                 
+     garbage-collector     -                 
      postgres             1                3
      moray                1                3
 --------------------------------------------------
@@ -725,7 +725,7 @@ summary:
      marlin               -               16
      reshard              -                 
      pgstatsmon           -                 
-     garbage-collector    -                 
+     garbage-collector     -                 
      postgres             1                3
      postgres             2                3
      postgres             3                3
@@ -1042,7 +1042,7 @@ summary:
      marlin               -               48
      reshard              -                 
      pgstatsmon           -                 
-     garbage-collector    -                 
+     garbage-collector     -                 
      postgres             1                3
      postgres             2                3
      moray                1                3
@@ -1527,7 +1527,7 @@ summary:
      marlin               -               96
      reshard              -                 
      pgstatsmon           -                 
-     garbage-collector    -                 
+     garbage-collector     -                 
      postgres             1                3
      postgres             2                3
      postgres             3                3
@@ -1741,7 +1741,7 @@ summary:
      marlin               -               16               16               16
      reshard              -                                                   
      pgstatsmon           -                                                   
-     garbage-collector    -                                                   
+     garbage-collector     -                                                   
      postgres             1                1                1                1
      moray                1                1                1                1
 --------------------------------------------------
@@ -1968,7 +1968,7 @@ summary:
      marlin               -               16               16               16
      reshard              -                                                   
      pgstatsmon           -                                                   
-     garbage-collector    -                                                   
+     garbage-collector     -                                                   
      postgres             1                1                1                1
      postgres             2                1                1                1
      moray                1                1                1                1
@@ -2436,7 +2436,7 @@ summary:
      marlin               -               48               48               48
      reshard              -                                                   
      pgstatsmon           -                                                   
-     garbage-collector    -                                                   
+     garbage-collector     -                                                   
      postgres             1                1                1                1
      postgres             2                1                1                1
      postgres             3                1                1                1
-- 
2.21.0

